internal/ignition/ignition.go

```diff
@@ -191,6 +196,30 @@ const highlyAvailableInfrastructureTopologyPatch = `---
 
 const tempNMConnectionsDir = "/etc/assisted/network"
 
+const bootReporterPath = "boot-reporter/assisted-boot-reporter.sh"
+
+var assistedBootReporterunitTemplate = `[Unit]
+Description=Collect and upload host boot logs to assisted-service
+Wants=network-online.target
+After=network-online.target
+DefaultDependencies=no
+[Service]
+Environment=ASSISTED_SERVICE_URL=%s
+Environment=PULL_SECRET_TOKEN=%s
```

Comment:
@tsorya why do we need this? how is the controller talking to the service? 
Can we use the same auth here?

Reply:
Nope, controller is getting pull secret from secret.